ui/components/app/permissions-connect-permission-list/permissions-connect-permission-list.js

```diff
@@ -0,0 +1,35 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+
+export default function PermissionsConnectPermissionList({ permissions }) {
+  const t = useI18nContext();
+
+  const PERMISSION_TYPES = useMemo(() => {
+    return {
+      eth_accounts: {
+        leftIcon: 'fas fa-eye',
+        label: t('eth_accounts'),
+        rightIcon: null,
+      },
+    };
+  }, [t]);
+
+  return (
+    <div className="permissions-connect-permission-list">
+      {Object.keys(permissions)
+        .map((permission) => PERMISSION_TYPES[permission])
+        .map((permission) => (
```

Comment:
I'd prefer that we didn't double `map`; it seems reasonable to use `PERMISSION_TYPES[permission].label`, etc.

Reply:
I agree with this ^